src/org/usfirst/frc/team467/robot/RobotMap.java

```diff
@@ -233,8 +233,8 @@ public static void init(RobotID id) {
 	public static final double ELEVATOR_HEIGHT_RANGE_INCHES = 94.5;
 	public static final double ELEVATOR_ERROR_TOLERANCE_INCHES = 1.0;
 
-	public static final int ELEVATOR_BOTTOM_TICKS = 812;
-	public static final int ELEVATOR_TOP_TICKS = 364;
+	public static final int ELEVATOR_BOTTOM_TICKS = 764;
```

Comment:
We should comment how we derived these values.

An observation, these values are similar in nature to the Stops heights in the elevator class. I think the stops heights probably should be moved to RobotMap, too. These will be different between comp1 and comp2.